src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx

```diff
@@ -0,0 +1,101 @@
+import 'react-date-range/dist/styles.css';
+import 'react-date-range/dist/theme/default.css';
+
+import {Calendar} from 'react-date-range';
+import React from 'react';
+import moment from 'moment';
+import styled from 'react-emotion';
+
+import {inputStyles} from 'app/styles/input';
+import DropdownMenu from 'app/components/dropdownMenu';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+import InputField from './inputField';
+
+export default class DatePickerField extends React.Component {
+  onChange = (onChange, onBlur, date) => {
+    onChange(date, {});
+    onBlur(date, {});
+  };
+
+  render() {
+    return (
+      <InputField
+        {...this.props}
+        field={({onChange, onBlur, value, disabled, name, id, ...props}) => {
+          const inputValue = !value ? new Date() : new Date(value);
```

Comment:
Could the value be invalid here?

Reply:
Addressed in https://github.com/getsentry/sentry/pull/13205